doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -349,26 +352,36 @@ Reserving notations
    .. note:: The notations mentioned in the module :ref:`init-notations` are reserved. Hence
              their precedence and associativity cannot be changed.
 
-   .. cmdv:: Reserved Infix "@symbol" {* @modifiers}
+   .. cmd:: Reserved Infix @string {? ( {+, @syntax_modifier } ) }
 
       This command declares an infix parsing rule without giving its
       interpretation.
 
    When a format is attached to a reserved notation, it is used by
    default by all subsequent interpretations of the corresponding
-   notation. A specific interpretation can provide its own format
-   overriding the default format though.
+   notation. Individual interpretations can override the format.
+
+   .. todo PR: this is talking about (A + B) vs (A + B)%nat, right?
+      wording could be better
```

Comment:
Right, although I don't know what a "format" is.

Reply:
No, this is not about `(A + B)` vs `(A + B)%nat`.  This is talking about things like
```coq
Reserved Notation "f +" (at level 50, format "f +").
```
where you can adjust the spacing and line-breaking of printing.